src/Compatibility/Microsoft.DotNet.ValidationSuppression/Microsoft.DotNet.ValidationSuppression.csproj

```diff
@@ -0,0 +1,12 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
I initially separated it since I considered it to be a concept separate to APICompat since theoretically it could be used by PackageValidation on it's own, so I tried to keep ApiCompat project to only have APICompat-specific things. That said, as discussed offline, I don't have a strong feeling about this so I'm fine if we decide to collapse even though in my opinion, the conceptual layering does ask for this concept to be separate.

Reply:
I think it makes sense to have it live the lowest on the stack as possible as API Compat will endup using this at some point when we add an MSBuild Task or dotnet global tool.